lib/OpenQA/Utils.pm

```diff
@@ -305,15 +306,25 @@ my %bugrefs = (
 );
 my %bugurls = reverse %bugrefs;
 
+sub bugref_regex {
+    my $regex = join('|', keys %bugrefs);
+    return qr/(($regex)#(\d+))(?=\s|$)/;
```

Comment:
One thing I noticed that you can't separate bug refs and urls by anything else then space or eol. I imagine comma, semicolon and dot should be also fine.